gradio/events.py

```diff
@@ -306,6 +306,63 @@ def __init__(self, target: Block | None, data: Any):
         """
 
 
+@document()
+class RetryData(EventData):
```

Comment:
Consider adding a `.value` attribute to `gr.RetryData` and `gr.UndoData` carrying the value of the message, in order to be consistent with `gr.LikeData`